_posts/2022-05-16-changelog-170.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #170"
+date: 2022-05-16 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#7849] Add inspections and quick-fixes for `unused_must_use` and `clippy::double_must_use` lints
+
+  <img src="/assets/posts/changelog-170/unused_must_use.png" width="600px"/>
```

Comment:
Maybe gif is better here to show some quick-fix in action as well? Also, we can show suppressing quick-fix as well

Reply:
oh, btw i've recorded a gif but forgot to include it into the PR
![Peek 2022-05-02 16-57--must-use-demo-1](https://user-images.githubusercontent.com/662976/168422641-6ac6c3a0-6fbe-4fbc-83d1-20ad99c1f0cf.gif)